README.md

```diff
@@ -219,35 +219,17 @@ To customize preact-cli create a `preact.config.js` or a `preact.config.json` fi
 ```js
 // ... imports or other code up here ...
 
-// these props are both optional
-export default {
-	// you can add preact-cli plugins here
-	plugins: [
-		// either a function
-		// (you'd probably import this because you can use the `webpack` function instead of an inline plugin)
-		function () {},
-		// strings also work (they get imported by preact-cli), useful for the json config
-		'plugin-name',
-		// with options
-		[
-			'plugin-name',
-			{
-				option: true,
-			},
-		],
-	],
-	/**
-	 * Function that mutates the original webpack config.
-	 * Supports asynchronous changes when a promise is returned (or it's an async function).
-	 *
-	 * @param {object} config - original webpack config.
-	 * @param {object} env - options passed to the CLI.
-	 * @param {WebpackConfigHelpers} helpers - object with useful helpers for working with the webpack config.
-	 * @param {object} options - this is mainly relevant for plugins (will always be empty in the config), default to an empty object
-	 **/
-	webpack(config, env, helpers, options) {
-		/** you can change the config here **/
-	},
```

Comment:
I think this format should be moved into the wiki, to keep it documented, but not the first thing people come across. I figure most beginners just want a simple set up to copy/paste, and this is a lot more busy than needs be.

Might just be me though? Would like thoughts on this.

Reply:
Yeah, you're probably right

not like we have a lot of plugins